### PR TITLE
fix import error on jinjaext when building docs

### PR DESCRIPTION
**Pull request by Ublimjo**

### CODE REVIEW

**Comment by Ublimjo** [comment]
#903

File: docs/jinjaext.py [hunk 1]
```diff
@@ -25,7 +25,8 @@
 from pygments.token import Keyword, Name, Comment, String, Error, \
      Number, Operator, Generic
 from jinja2 import Environment, FileSystemLoader
-from jinja2._compat import abc
+from collections import abc
+# from jinja2._compat import abc
```
Comment (ThiefMaster) [inline docs/jinjaext.py]: please don't add commented-out code

**Comment by deoren** [comment]
It's not pretty, but this quick hack let me build the docs in epub format a few minutes ago:

```diff
diff --git a/docs/jinjaext.py b/docs/jinjaext.py
index fd38ee8..10a495e 100644
--- a/docs/jinjaext.py
+++ b/docs/jinjaext.py
@@ -25,7 +25,7 @@ from pygments.style import Style
 from pygments.token import Keyword, Name, Comment, String, Error, \
      Number, Operator, Generic
 from jinja2 import Environment, FileSystemLoader
-from jinja2._compat import abc
+#from jinja2._compat import abc


 def parse_rst(state, content_offset, doc):
@@ -160,7 +160,7 @@ def jinja_nodes(dirname, arguments, options, content, lineno,
             members = []
             for key, name in node.__dict__.items():
                 if not key.startswith('_') and \
-                   not hasattr(node.__base__, key) and isinstance(name, abc.Callable):
+                   not hasattr(node.__base__, key):
                     members.append(key)
             if members:
                 members.sort()
```

FWIW, Google Play Books seems OK with the resulting epub (so far at least).

**Comment by davidism** [comment]
See comment on linked issue, this will be addressed as I switch to pallets-sphinx-themes.

**Pull request closed without merging**